Analyzer/src/AnalyzeDoubleDisCo.cc

```diff
@@ -1623,8 +1645,12 @@ void AnalyzeDoubleDisCo::Loop(NTupleReader& tr, double, int maxevents, bool isQu
                                     //  -- each of A, B, C, D has 7,8,9,10,11, 12incl Njets bins
                                     // ---------------------------------------------------------
                                     if (njets == "Incl")
-                                        my_histos["h_njets_12incl" + name]->Fill(theGoodJets>=12 ? 12-theMVAjets+shift*6 : theGoodJets-theMVAjets+shift*6, w);
-
+                                    {
+                                        //theMVAjets This is basically hardcoded to be 7 always. However, in the future it shouldn't and the following lines need to be updated then
```

Comment:
So, yes, it essentially hardcoded, because we elect to instantiate once instance of `MakeMVAVariables`, and seven is the "magic" number that we pass for number of jets. I am trying to think of a better way to setup that variable. I guess we could have an "official" number of jets variable for each channel that is wholly separate from `MakeMVAVarialbles`. Perhaps we just set these hypothetical "official" number of jets variables somewhere in `Config.h` (?) Then they would be a bit more visible/obvious to the user.

Reply:
Ok. Cool. For now I will remove this from the money plot based on our discussion.